packages/palette/src/elements/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC } from "react"
+import { Box, Flex, breakpoints } from "@artsy/palette"
+import styled, { css } from "styled-components"
+import { zIndex } from "styled-system"
+import { useDrawer } from "./useDrawer"
+
+export interface DrawerProps {
+  anchor?: "left" | "right"
+  flexContent?: number
+  flexOverlay?: number
```

Comment:
any recommendation for this proportion case? Or we can hardcode the flex value proportion until we have a DS official size?

Reply:
I'd imagine the size is entirely dependent on the content? So that might change depending on use case. We're going to be using this for the filters for instance, which will be pretty specific. Just setting defaults that are over-writable with `BoxProps` is the way to go. I just read these props and have no idea what they mean really. We could have specific `contentProps` and `overlayProps` slots that each accept `BoxProps` is also fine here.